iconsets/rollup-iconsets.js

```diff
@@ -0,0 +1,7 @@
+// This file is the entry file for css rollup
```

Comment:
This comment seems to conflict with the filename. The filename is telling me that "This file is the entry file for rolling up iconsets", not CSS in general.

Which was your intention and could you update the filename/this comment accordingly?

Reply:
Fixed in https://github.com/suda/tool-bar/pull/302/commits/a717e6a6854644b9ccb0a791bc6e181fe97bded6